config/routes.rb

```diff
@@ -27,11 +27,13 @@
     resources :ratings, only: :create
     resources :change_requests, only: :create
     resources :services, only: :create
+    resources :feedbacks, only: [:create, :index]
   end
   resources :services do
     resources :ratings, only: :create
     resources :change_requests, only: :create
     resources :notes, only: :create
+    resources :feedbacks, only: :create
```

Comment:
wouldn't we want a finder for feedbacks per service as well?

Reply:
definitely I missed to add it on services, thank you